docs/content/preview/launch-and-manage/monitor-and-alert/metrics/throughput.md

```diff
@@ -12,26 +12,60 @@ menu:
 type: docs
 ---
 
-YugabyteDB supports additional attributes for latency metrics, which enable you to calculate throughput.
-
-These attributes include the following:
+As referenced in the earlier section, YugabyteDB has latency metrics for Tables/Tablets available on port `9000` for Tserver and port `7000` for master. Additionally, YugabyteDb supports query processing and connection metrics on port `13000` for YSQL and on port `12000` for YCQL. Table/Tablet metrics on Master and Tserver have different set of attributes, which enable you to calculate throughput as compared to query processing and connection metrics for YSQL and YCQL. These are described below. 
+
+Latency metrics for Tables and Tablets in JSON format, for example, `handler_latency_yb_tserver_TabletServerService_Read` - latency metrics to perform READ operation at a tablet level, will have this body 
+```json
+{
+  "name": "handler_latency_yb_tserver_TabletServerService_Read",
+  "total_count": 14390,
+  "min": 104,
+  "mean": 171.65,
+  "percentile_75": 0,
+  "percentile_95": 0,
+  "percentile_99": 0,
+  "percentile_99_9": 0,
+  "percentile_99_99": 0,
+  "max": 1655,
+  "total_sum": 2470154
+}
+```
+
+And will include the following attributes:
 
 | Attribute | Description |
 | :--- | :--- |
-| `total_count/count` | The number of times the value of a metric has been measured.
-| `min` | The minimum value of a metric across all measurements.
-| `mean` | The average value of the metric across all measurements.
-| `Percentile_75` | The 75th percentile value of the metric across all measurements.
-| `Percentile_95` | The 95th percentile value of the metric across all measurements.
-| `Percentile_99` | The 99th percentile of the metric across all metrics measurements.
-| `Percentile_99_9` | The 99.9th percentile of the metric across all metrics measurements.
-| `Percentile_99_99` | The 99.99th percentile of the metric across all metrics measurements.
-| `max` | The maximum value of the metric across all measurements.
-| `total_sum/sum` | The aggregate of all the metric values across the measurements reflected in total_count/count.
+| `total_count` | The number of times the latency of a metric has been measured.
+| `min` | The minimum value of the latency for the metric across all measurements.
+| `mean` | The average latency for the metric across all measurements.
+| `Percentile_75` | The latency at which 75% of the requests were completed successfully for the metric.
```

Comment:
at which --> under which

or, reword to something like:

The 75th percentile latency across all measurements of this metric.